components/docs-chef-io/content/automate/node_integrations.md

```diff
@@ -156,7 +156,7 @@ We do not support Azure Government Cloud.
 When creating an Azure VM Node Manager, you will be required to provide:
 
 1. A name for your manager
-1. Your Azure credentials (client ID, client secret, and tenant ID)
+2. Your Azure credentials (client ID, client secret and tenant ID)
```

Comment:
add Subscription ID

Reply:
we don’t need to add a Subscription ID here, Subscription ID will be support for azure-api not for VM

Reply:
ok

Reply:
```suggestion
2. Your Azure credentials (client ID, client secret, and tenant ID)
```

Unless the user needs to enter the client secret and the tenant ID _together_, then this sentence needs a serial comma.

Reply:
@shaik80 If you make these changes, then we can merge this PR.

Reply:
@vivek-yadav I need to un-resolve this conversation so the changes are visible.